### PR TITLE
Changes to avoid compiler segmentation fault

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -41,5 +41,5 @@ script:
          cd ios/Framework
          pod _0.39.0_ install
          set -o pipefail
-         ./run-tests.sh
+         xcodebuild clean test -workspace LiferayScreens.xcworkspace -scheme LiferayScreens -destination "platform=iOS Simulator,name=iPhone 6 Plus" | xcpretty
```
Comment (jmnavarro) [inline .travis.yml]: why this?

File: ios/Framework/Core/Auth/UserPortraitScreenlet/Interactors/DownloadUserPortraitInteractor.swift [hunk 2]
```diff
@@ -243,9 +243,9 @@ class DownloadUserPortraitInteractor: ServerReadConnectorInteractor {
 		let portraitEntry = attributes?["portraitId"]
 		let userEntry = attributes?["userId"]
 		if let attributes = attributes,
-				portraitId = portraitEntry?.description.asLong,
+				portraitId = portraitEntry?.longLongValue,
```
Comment (jmnavarro) [inline ios/Framework/Core/Auth/UserPortraitScreenlet/Interactors/DownloadUserPortraitInteractor.swift]: if `asLong()` doesn't make sense anymore, just remove it and use your solution everywhere

**Comment by victorg1991** [comment]
Reopen on https://github.com/liferay/liferay-screens/pull/338

File: .travis.yml (hunk 1 above)
Comment (victorg1991) [inline .travis.yml]: 👍 0364b31

File: ios/Framework/Core/Auth/UserPortraitScreenlet/Interactors/DownloadUserPortraitInteractor.swift (hunk 2 above)
Comment (victorg1991) [inline ios/Framework/Core/Auth/UserPortraitScreenlet/Interactors/DownloadUserPortraitInteractor.swift]: 👍 	1b01210

**Pull request closed without merging**